dashboard/test/ui/features/acquisition_products/pl_landing_page.feature

```diff
@@ -88,6 +87,7 @@ Feature: Professional Learning landing page
 
   Scenario: Workshop Organizer sees relevant content sections
     Given I am an organizer with started and completed courses
+    And I wait for 2 seconds
```

Comment:
is there an element on the page that we can wait for instead of always waiting 2 seconds?

Reply:
I don't think so since we haven't sent the test to look at any particular page yet. This test was frequently failing when it tried to load the page on line 91 (saying it either couldn't find a user object or the user didn't have permissions). Since adding a wait time fixed it, my guess is that it was loading the page as the previous step was still finishing creating the test user and assigning them the `workshop_organizer` permission so there was a bit of a race between those two lines.

Reply:
Oh that's fair. What if I mirror [this test's structure](https://github.com/code-dot-org/code-dot-org/pull/62758/files/9fc1e2b63d574aeb9dbc9e1bb171c13ff793fe84#diff-2317701a7aad24bdc8bec04ea731c48214bafd86fc565f09956902328ada826bR6-R8) and have signing in as part of it so that it'll be easier to tell where it's failing. So if it fails at sign-in we know that it failed to have a user ready or if it fails trying to load the right My PL tab then we know it failed to have the user's permissions set up correctly and in time. Then it won't have an arbitrary wait time and (should this test be flaky in the future) we can debug it much more easily.

Reply:
Unfortunately this didn't work and I spent all of yesterday trying to find a more effective solution but to no avail. I think to timebox this change and to get out something that will at least make the test less flaky, I'm inclined to call it good here and revisit with more investigation if this test becomes flaky again if that sounds okay?

Reply:
yeah, I agree. there are so many places where we've already relied on waiting a set number of seconds this isn't going to make a huge difference. definitely appreciate the effort on this!